modules/infrastructure-moving-monitoring.adoc

```diff
@@ -5,10 +5,65 @@
 [id="infrastructure-moving-monitoring-{context}"]
 = Moving the monitoring solution
 
-[IMPORTANT]
-====
-The monitoring solution uses Cluster Version Operator (CVO) to create the
-ConfigMap that the monitoring Operator uses to determine how to deploy its
-resources. Because it uses the CVO and users cannot modify the CVO, you cannot
-change where the Operator deploys resources.
-====
\ No newline at end of file
+
+By default, a Prometheus, Grafana, and AlertManager stack is deployed to
+provide cluster monitoring. It is managed by the Cluster Monitoring Operator,
+and to move its components to different machines, you apply a custom ConfigMap.
+
+.Procedure
+
+. Optionally, review the default ConfigMap for the monitoring stack:
++
+----
+$ oc get configmap cluster-monitoring-config -n openshift-monitoring
+----
+
+. Save the following ConfigMap definition as the
+`cluster-monitoring-configmap.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |+
+    alertmanagerMain:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    prometheusK8s:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    prometheusOperator:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    grafana:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    k8sPrometheusAdapter:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    kubeStateMetrics:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+    telemeterClient:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+----
++
+Running this ConfigMap forces the components of the monitoring stack to redeploy
+to infrastructure nodes.
+
+. Apply the new ConfigMap:
++
+----
+$ oc create cluster-monitoring-configmap.yaml
```

Comment:
should be "oc create -f cluster-monitoring-configmap.yaml"

Note: we assume the cluster-monitoring-config configmap is not exist